action-entrypoint.py

```
@@ -0,0 +1,387 @@
+
+def main(argv: Sequence[str]) -> int:
+    inputs = parse_inputs(argv)
+    workspace = Path(os.environ.get("GITHUB_WORKSPACE", os.getcwd())).resolve()
+    working_dir = within_workspace(
+        workspace,
+        inputs.working_directory,
+        "working-directory",
+        directory=True,
+    )
+    command = build_command(inputs)
+    completed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)